R/database.R

```diff
@@ -2717,15 +2679,16 @@ get_manta_sv_by_samples = function(these_samples_metadata,
 #' This function is used for retrieving Manta results (structural variants) from individual flat-files (one sample). 
 #' For multiple samples, please see get_manta_sv_by_samples (a convenience wrapper function for get_manta_by_sample). 
 #' Additional columns are extracted from the VCF column and standard filtering options are available. 
-#' This function also performs a lift-over to selected projection, if needed.   
+#' This function also performs a lift-over to selected projection, if needed. 
+#' Please note, if force_lift is set to FALSE, an extra column will be added that states if the returned variant calls need to be lifted. 
+#' The value for this column is returned TRUE (for all rows) if the available genome projection for the selected sample does not match the selected projection (i.e requiring the user to manually lift the calls).
 #'
 #' @param this_sample_id The single sample ID you want to obtain the result from.
-#' @param these_samples_metadata A metadata table containing metadata for this_sample_id, or sample of interest. This parameeter is required.
-#' @param force_lift If TRUE, coordinates will be lifted (if needed) to the selected projection. WARNING! Should always be TRUE if this function is running in stand-alone mode.
+#' @param these_samples_metadata A metadata table containing metadata for this_sample_id, or sample of interest. This parameter is required.
+#' @param force_lift If TRUE, coordinates will be lifted (if needed) to the selected projection. Default is FALSE. WARNING: if running this function in stand-alone mode, set this parameter to TRUE to ensure that the returned calls are in respect to the requested projection.
```

Comment:
What is stand-alone mode? Do you mean "if your code calls this function directly"?

Reply:
Yes, that is what I meant. I have changed it to be more descriptive.